implement-shell-tools/ls/ls.py

```diff
@@ -0,0 +1,59 @@
+import argparse

```

Comment:
Please also test against the examples in the readme - some of them don't work the same.